tests/opkg.py

```diff
@@ -36,8 +36,23 @@ def runTest(self):
 class OpkgUpdate(rootfs_boot.RootFSBootTest):
     '''Opkg is able to update list of packages.'''
     def runTest(self):
-        board.sendline('\nopkg update && echo "All package lists updated"')
+        board.sendline("\nopkg update | tee opkg.txt && ! grep 'Signature check failed' opkg.txt"
+                        "&& echo 'All package lists updated' && rm opkg.txt")
         board.expect('Updated list of available packages')
         board.expect('All package lists updated')
         board.expect(prompt)
 
+    def recover(self):
+        board.sendline('\nrm opkg.txt')
+        board.expect(prompt)
+
+class OpkgInstall(rootfs_boot.RootFSBootTest):
+    '''Opkg is able to install selected packages'''
+    def runTest(self):
+        # One package per feed: packages, openwrt-routing, openwrt-managements,
+        # ci40-platform-feed, telephony, luci
+        packages = [ "nano", "mrd6", "shtool", "glog", "miax", "luci-mod-rpc" ]
```

Comment:
That's been mentioned before, the issue is that when building with latest packages and such this might fail the test due to kernel mismatch.